plugins/finder/jevents/jevents.php

```diff
@@ -384,19 +342,121 @@ protected function getItem($id)
 	 */
 	protected function getStateQuery()
 	{
-		$sql = $this->db->getQuery(true);
+		$db     = $this->db;
+		$query  = $db->getQuery(true);
+
```

Comment:
If you're talking about getStateQuery() method as a whole I think it is. The default line of code is:

// Item ID
$query->select('a.id');

that will return an error since neither of JEvents _vevent or _vevdetails are using id as column names.